File: src/resources/bank-account.js
db = connect("localhost:27017/app17");
coll = db.cars;
coll.insert({
    "bankName" : "Chase",
    "accountNumber" : Number(098172304987123),
    "routingNumber" : Number(213947293),
    "accountHolderName" : "Sue Suzzie Sizzle",
    "verified" : false});

File: src/main/java/app17/rest/BankAccount.java
package app17.rest;


public class BankAccount {
    String id = null;
    String bankName, accountHolderName;
    Number accountNumber, routingNumber;
    Boolean verified;

    public BankAccount(String bankName, Number accountNumber, Number routingNumber, String accountHolderName,
                       Boolean verified) {

        this.bankName = bankName;
        this.accountNumber = accountNumber;
        this.routingNumber = routingNumber;
        this.accountHolderName = accountHolderName;
        this.verified = verified;
    }
    public void setId(String id) {
        this.id = id;
    }
}
